### ISSUE TITLE
Stopped working 

### ISSUE THREAD

**Issue by Mightytech0** [gilesgc/Freemote#9]
Emotes stopped working as of today appears in blank box when typing emote. @gilesgc  #7

**Comment by liljesusbeatz** [comment]
yeah, twitch added animated emotes on iOS and now freemote doesnt work anymore.
i had to downgrade to twitch version 12.0 to make it work again
